samples/bluetooth/peripheral_bms/README.rst

```diff
@@ -0,0 +1,102 @@
+.. _peripheral_bms:
+
+Bluetooth: Peripheral Bond Management Service (BMS)
+###################################################
+
+The peripheral BMS sample demonstrates how to use the :ref:`bms_readme`.
+
+Overview
+********
+
+When connected, the sample waits for Client requests to perform any bond deleting operation.
+
+It supports up to two simultaneous Client connections.
+
+Requirements
+************
+
+* One of the Nordic development boards:
+
+  * |nRF5340DK|,
+  * |nRF52840DK|,
+  * |nRF52DK|.
+
+* A Bluetooth Low Energy dongle or development kit.
+
+* nRF Connect for Desktop.
```

Comment:
no full stop here or in line 24, these are not sentences (this one is just a product name)

Reply:
fixed